src/main/java/chess/dao/ChessGameDao.java

```diff
@@ -0,0 +1,101 @@
+package chess.dao;
+
+import chess.converter.PieceConverter;
+import chess.domain.ChessGame;
+import chess.domain.piece.Piece;
+import chess.domain.position.File;
+import chess.domain.position.Position;
+import chess.domain.position.Rank;
+import chess.dto.ChessGameDto;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class ChessGameDao {
+
+    private JdbcTemplate jdbcTemplate;
```

Comment:
```suggestion
    private final JdbcTemplate jdbcTemplate;
```
런타임에 jdbctemplate이 변경되지 않도록 final을 붙이면 좋을 것 같습니다~

Reply:
의존성 주입에 대해서 저랑 페어 둘 다 잘 모르는 상태였어서..필드 주입을 시도하다보니 final이 누락되었던 것 같아요..!!

필드 주입 대신 생성자 주입을 사용한 이유는 다음 블로그 글을 참고하였습니다!!
[생성자 주입을 @Autowired를 사용하는 필드 주입보다 권장하는 하는 이유](https://madplay.github.io/post/why-constructor-injection-is-better-than-field-injection)